src/resources/Ingresses/IngressDetails.js

```diff
@@ -1,39 +1,54 @@
 import { useTranslation } from 'react-i18next';
 
 import { ResourceDetails } from 'shared/components/ResourceDetails/ResourceDetails';
-import { EMPTY_TEXT_PLACEHOLDER } from 'shared/constants';
 
 import { Rules } from './Rules';
 import { DefaultBackendPanel } from './DefaultBackendPanel';
 import IngressCreate from './IngressCreate';
 import { ResourceDescription } from 'resources/Ingresses';
+import { EventsList } from 'shared/components/EventsList';
+import { filterByResource } from 'hooks/useMessageList';
+import { StatusBadge } from 'shared/components/StatusBadge/StatusBadge';
+import { IngressStatus } from './IngressStatus';
+import { IngressSpecification } from './IngressSpecification';
+import { useState } from 'react';
 
 export function IngressDetails(props) {
   const { t } = useTranslation();
+  const [totalPods, setTotalPods] = useState(0);
+  const [healthyPods, setHealthyPods] = useState(0);
 
-  const getLoadBalancer = ingress => {
-    if (ingress.status.loadBalancer?.ingress) {
-      return ingress.status.loadBalancer?.ingress
-        .map(endpoint => endpoint.ip || endpoint.hostname)
-        .join(', ');
-    } else {
-      return EMPTY_TEXT_PLACEHOLDER;
-    }
+  const calculateTotalPorts = ingress => {
+    let allPorts = 0;
+
+    ingress?.status?.loadBalancer?.ingress?.forEach(element => {
+      element?.ports?.forEach(() => allPorts++);
+    });
+
+    setTotalPods(allPorts);
   };
 
-  const customColumns = [
-    {
-      header: t('ingresses.labels.load-balancers'),
-      value: getLoadBalancer,
-    },
-    {
-      header: t('ingresses.labels.ingress-class-name'),
-      value: ingress => ingress.spec.ingressClassName,
-    },
-  ];
+  const calculatePortsWithoutErrors = ingress => {
+    let healthyPods = 0;
+
+    ingress?.status?.loadBalancer?.ingress?.forEach(element => {
+      element?.ports?.forEach(port => {
+        if (!port.error) healthyPods++;
+      });
+    });
+
+    setHealthyPods(healthyPods);
+  };
```

Comment:
we can merge these two methods, using a single loop for both

Reply:
No, it was a typo :)